map/cMapEditor.cpp

```diff
@@ -1,5 +1,8 @@
+#include "../include/d2tmh.h"
```

Comment:
Nope. We're trying to get rid of this header.

Also, always include the corresponding header first. Then the compilation run checks if that header can be included without caring about include order.

Reply:
Not me this include.
It will be here because code change.